extensions/dicom-rt/src/components/LoadingIndicator/LoadingIndicator.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
@dannyrb Is this not a component somewhere we can just import instead of copying here?

Reply:
Linked wrong line, this whole file I mean.

Reply:
@JamesAPetts Although they share the same name this component is totally different than the one in the UI project. This is an animated SVG of a loading indicator that I think it's better than the icon we have today.  Currently, the Icon component doesn't work with animated SVGs. @dannyrb what do you think about having this one as the new OHIF default?

Reply:
@dannyrb Poking you for this, although I'm not bothered about it for this particular PR.